micro-osd.sh

```diff
@@ -43,6 +43,7 @@ auth cluster required = none
 auth service required = none
 auth client required = none
 osd pool default size = 1
+mon data avail crit = 2
```

Comment:
This change does not seem to be needed. The filesystem where I run my tests is too full!